src/applications/simple-forms/form-upload/routes.jsx

```diff
@@ -1,23 +1,34 @@
 import { createRoutesWithSaveInProgress } from 'platform/forms/save-in-progress/helpers';
+import environment from '~/platform/utilities/environment';
 import formConfig from './config/form';
-import App from './containers/App.jsx';
+import App from './containers/App';
+
+// Add any new form-upload forms to this list
+const formUploadForms = ['21-0779'];
 
 const config = formConfig();
+const baseUrl = environment.BASE_URL.replace(/3001/g, '3002');
+const indexRouteByForm = formId => ({
+  onEnter: () => {
+    window.location.href = `${baseUrl}/find-forms/about-form-${formId}/`;
+  },
+});
 
-const routes = [
-  {
-    path: '/21-0779',
+const routes = formUploadForms.map(formId => {
+  return {
+    path: `/${formId}`,
     component: App,
-    indexRoute: { onEnter: (nextState, replace) => replace('/introduction') },
+    indexRoute: indexRouteByForm(formId),
```

Comment:
Do we want the index route to be the find-a-form page (as in here) or Step 1 of the flow itself (the upload step)? I thought we wanted it to be the first step of the flow but I may be misremembering.